app/src/main/java/com/parishod/watomatic/fragment/DonationFragment.kt

```diff
@@ -39,9 +41,24 @@ class DonationFragment : Fragment() {
         fragmentView.paypal.setOnClickListener {
             launchUrl("https://www.paypal.com/paypalme/deek")
         }
+
+        fragmentView.bitcoin.setOnClickListener {
+            launchBitcoin()
+        }
         return fragmentView
     }
 
+    private fun launchBitcoin(){
+        val i = Intent(Intent.ACTION_VIEW)
+        i.data = Uri.parse("bitcoin:bc1qv6zmgw845pktq9jr5qztup80qufu0yg46ur2kw")
+
+        try {
+            startActivity(i)
+        } catch (e: ActivityNotFoundException) {
+            launchUrl("https://www.blockchain.com/btc/address/bc1qv6zmgw845pktq9jr5qztup80qufu0yg46ur2kw")
```

Comment:
Can we use address from a variable instead of hardcoding so we can do not forget to change it later when we update the address?

(Ideally these including paypal and liberay links can go to strings.xml I think with translations set to false but we can do that later too)

Reply:
moved those to constants